website/content/docs/platform/k8s/helm/enterprise.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Running Vault Enterprise - Kubernetes
+description: >-
+  Vault Helm supports deploying Vault Enterprise, including license autoloading.
+---
+
+# Vault Enterprise
+
+You can use this Helm chart to deploy Vault Enterprise by following a few extra steps around licensing.
+
+~> **Note:** As of Vault Enterprise 1.8, the license must be specified via HCL configuration or environment variables on startup, unless the Vault cluster was created with an older Vault version and the license was stored, or the Vault Enterprise binary has the license baked in. More information is available in the [Vault Enterprise License docs](/docs/enterprise/license).
```

Comment:
The Vault Enterprise License docs are going up in https://github.com/hashicorp/vault/pull/11809, so this link isn't live yet.